railties/test/generators/model_generator_test.rb

```diff
@@ -345,26 +345,6 @@ def test_check_class_collision
     assert_match(/The name 'Object' is either already used in your application or reserved/, content)
   end
 
-  def test_index_is_added_for_belongs_to_association
-    run_generator ["account", "supplier:belongs_to"]
-
-    assert_migration "db/migrate/create_accounts.rb" do |m|
-      assert_method :change, m do |up|
-        assert_match(/index: true/, up)
-      end
-    end
-  end
-
-  def test_index_is_added_for_references_association
-    run_generator ["account", "supplier:references"]
-
-    assert_migration "db/migrate/create_accounts.rb" do |m|
-      assert_method :change, m do |up|
-        assert_match(/index: true/, up)
-      end
-    end
-  end
-
```

Comment:
I removed these tests. Should I add them for checking `index_is_not_added_for_references_association` ?